website/docs/platform/5-accessibility.md

```diff
@@ -0,0 +1,165 @@
+
+## Focus management
+
+Focus lets users know what they can interact with.
+A cocused element typically has a special style, such as a bold blue ring around them. The ability to jump between and identify focusable elements is critical, especially for keyboard-only users and people who use assistive tech.
+
+Bring focus to an element:
+
+```js
+export default test('Email input')
+  .step(Page.visit('/'))
+  .step(TextField('Email').focus())
+```
+
+Find the input element that has focus, using the filter `{ focused: true }`:
+
+```js
+export default test('Find focused input')
+  .step(Page.visit('/'))
+  .step(TextField('Email').focus())
+  .step(
+    TextField({ focused: true })
+      .fillIn('something@example.com')
+  )
+```
+
+Assert that an element has focus:
+
+```js
+export default test('Focus')
+  .step(Page.visit('/'))
+  .step(TextField('Email').focus())
+  .assertion(
+    TextField('Email').is({ focused: true })
+  )
+```
+
+Focus is just another property of an HTML element. Therefore, when testing for focus you can use an approach that is similar to how you would test other elements, such as whether a checkbox is checked or a button is enabled.
+
+## Alt text
+
+Every image in your app that has important meaning should have an alt property that describes it.
+
+Write an interactor for images that finds them by their alt text:
+
+```js
+const Image = createInteractor('image')
+  .selector('img')
+  .locator((element) => element['alt'])
+```
+
+Use this new interactor to find specific images:
+
+```js
+export default test('Image')
+  .step(Page.visit('/'))
+  .assertion(
+    Image('here is some alt text').exists()
+  )
+```
+
+Again, this is a lot like testing any other property in the DOM.
+In order for the test to succeed, the image must exist, which is useful to test on its own.
+In this way, accessibility checks can be woven right into other feature tests without increasing the lines of code.
+
+## Checking ARIA properties
+
+[ARIA properties](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques) provide additional information to a browser's accessibility API, such as whether a menu is open or closed. ARIA is short for Accessible Rich Internet Applications. ARIA properties do not cause a change in the page's DOM, but they do have an effect on the experience of people who use assistive technology.
+
+Checking if a menu is opened or closed, according to the [`aria-expanded` property](https://www.w3.org/WAI/tutorials/menus/application-menus/):
+
+```html
+<button aria-expanded="false">
+  Toggle
+</button>
+```
+
+Write an interactor that looks for the buttons that control toggling the open
+and close of a menu:
+
+```js
+export const Toggle = createInteractor('toggle')
+  .selector('button')
+  .locator((element) => element.innerHTML)
+  .filters({
+    ariaExpanded: (element) => element.ariaExpanded
+  })
```

Comment:
It will probably be easier to work with this filter if we convert it into a first-class boolean value
```suggestion
  .filters({
    ariaExpanded: (element) => element.ariaExpanded === "true"
  })
```